src/parser.ts

```diff
@@ -204,6 +206,15 @@ class Parser {
       }
     }
 
+    // Skip over PropTypes that are exported
+    if (
```

Comment:
I tried to do some checking that the type comes from `@types/prop-types`, but I couldn't figure it out.